### DISCUSSION TITLE
Beatmap failed to load error

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22185] [Q&A]
How can I get rid of it?
![image](https://user-images.githubusercontent.com/86934170/212394938-d71054d7-25ca-4160-be4e-072ac19774a4.png)

This two intro songs cause them
![image](https://user-images.githubusercontent.com/86934170/212395415-012e0914-01bb-4683-95fa-6d93dc5cc9f0.png)

[updater.log](https://github.com/ppy/osu/files/10414481/updater.log)
[database.log](https://github.com/ppy/osu/files/10414476/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10414477/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10414478/network.log)
[performance.log](https://github.com/ppy/osu/files/10414479/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10414480/runtime.log)

**Comment by bdach** [comment]
I'm more interested if you can try and provide context as to how this can happen in the first place. When did this start?

**Reply by Loreos7** [reply]
I don't remember exactly. I just started to experience this errors at some point. One or two months ago.
I didn't do anything with my database either.

**Reply by bdach** [reply]
Well we don't really have a facility to fix this since I've never seen this happen before. We may add one to perform integrity checks but right now probably the best bet to fix is a reinstall / copying over the missing files from another install somewhere else.

**Reply by Loreos7** [reply]
Alright, I thought about it. Thank you.

**Comment by peppy** [comment]
Sounds like your whole file store is gone. If you want to fix it the easiest way is to delete `client.realm` and start from scratch. Of course this will nuke your local beatmap collection.
